app.gummi.gummi.yml

```diff
@@ -0,0 +1,90 @@
+id: app.gummi.gummi
+runtime: org.gnome.Platform
+runtime-version: '43'
+sdk: org.gnome.Sdk
+command: gummi
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.texlive
+
+add-extensions:
+  org.freedesktop.Sdk.Extension.texlive:
+    directory: texlive
+    subdirectories: true
+    autodelete: true
+    version: '22.08'
+
+finish-args:
+  - --share=ipc
+  - --filesystem=xdg-documents
```

Comment:
you never display anything?

Reply:
Resolved this at https://github.com/flathub/flathub/pull/3789/commits/edb64ebf22a48ab22d9f185cda083e2180a13aec